react/features/video-menu/components/web/RemoteVideoMenuTriggerButton.tsx

```diff
@@ -302,13 +303,12 @@ function _mapStateToProps(state: IState, ownProps: Partial<Props>) {
         _disabled: remoteVideoMenu?.disabled,
         _menuPosition,
         _overflowDrawer: overflowDrawer,
-        _participant: participant,
-        _participantDisplayName,
+        _participant: participant ?? { id: '' },
```

Comment:
nit: from what I see participant can only be undefined or object. Think `participant || { id: '' }` would do here.

Reply:
Why would we use `||` instead of `??`? `??` returns the right side if the left is either `null` or `undefined`

Reply:
so is || . was thinking that the null coalescence is normally used when we expect cases when the left side value could actually be an empty string or 0, but it's never the case here. This is why it's a nit, it does work as expected but it might be just a bit confusing as it might suggest that we can expect 0 or '' in certain cases, even though we don't  :D

Reply:
Arguably, `??` is more correct, because `||` just works because the left side is coerced into a boolean. By modern standards, `??` is more "correct".